src/loaders/workspace_uploader/workspace_uploader_helper.py

```diff
@@ -0,0 +1,153 @@
+import os
```

Comment:
This is extremely similar to workspace_downloader_helper - is there a reason not to merge them?

Reply:
1. Is not really downloader helper no more, is uploader
2. In constructor part, I saw more differences than similarities. There are instance vars unique to uploader or downloader only.

We can merge but then we need to rename and relocate.

Reply:
We should merge and make this general use callback server runner code. I'd do the rename & relocate in a separate PR and then merge into this PR from main

Reply:
I prefer to do it in a separate PR after merging this one. Is it okay?

Reply:
No, I want to be able to see the changes made to the callback server wrapper for this change. Anyway, I'm not sure why making a separate PR and merging it now is any different than making a separate PR and merging it later

Reply:
https://github.com/kbase/collections/pull/555